spice/tests/rv_copypaste.py

```diff
@@ -77,5 +77,3 @@ def run(vt_test, test_params, env):
 
     if cfg.negative and success or not cfg.negative and not success:
         raise utils.SpiceTestFail(test, "Test failed.")
```

Comment:
"pass" was a  flag that test passes.     not for python, but rather for developer.

Maybe add something comment:
# test passes